packages/realm/bindgen/src/realm_js_jsi_helpers.h

```diff
@@ -134,7 +134,7 @@ REALM_NOINLINE inline jsi::Value toJsiException(jsi::Runtime& env, const std::ex
 
 [[noreturn]] REALM_NOINLINE inline void throwNullSharedPtrError(jsi::Runtime& env, const char* clsName)
 {
-    throw jsi::JSError(env, util::format("Attempting to use an instanace of $1 holding a null shared_ptr. "
+    throw jsi::JSError(env, util::format("Attempting to use an instanace of %1 holding a null shared_ptr. "
```

Comment:
I don't know why by brain wants to use `$1` placeholders rather than `%1` with `util::format`, but I seem to make this typo a lot...

Reply:
I have typed `$` so many times that I know I have to stop and think